src/pages/home/sidebar/SidebarLinksData.js

```diff
@@ -334,4 +330,22 @@ export default compose(
             initialValue: {},
         },
     }),
-)(SidebarLinksData);
+)(
+    memo(
```

Comment:
Based on how this looks like it'd be worth to comment it well in code - what these checks represent, why we need them so that it's easier to remove this once we solve the root cause later on.